### ISSUE TITLE
4.3.1 is not rendering for me

### ISSUE THREAD

**Issue by beccasaurus** [valentine195/obsidian-admonition#37]
Left is editor, right is preview:

![Screenshot from 2021-05-27 10-48-09](https://user-images.githubusercontent.com/280/119873301-42a19b00-bed9-11eb-9e92-0f9a61693e3b.png)

I am not 100% whether I was on 4.3.0 or 4.2.1 yesterday but everything was working fine. I tried uninstalling/re-installing as well as switching themes etc. No longer rendering for me anymore.

`Obsidian v0.12.13`

**Comment by valentine195** [comment]
Is there an error in the console (<kbd>Ctrl</kbd>-<kbd>Shift</kbd>-<kbd>I</kbd>)?

**Comment by beccasaurus** [comment]
> \* _Checking..._ \* nope.

I tried the 4.3.0 and 4.2.1 releases manually and it still didn't work.

Turns out, closing and re-opening the preview panel worked `<---` User error.

Sorry for bothering you! I saw `4.3.1` had just been updated + my Admonition was not rendering and jumped to conclusions after I couldn't get it to work.

(I'm loving the plugin, thank you for your work on it and _insanely fast response times_) \* _Closing_ \*
